streams/src/test/java/org/apache/kafka/streams/processor/internals/DefaultStateUpdaterTest.java

```diff
@@ -286,6 +308,7 @@ public void shouldRestoreActiveStatefulTasksAndUpdateStandbyTasks() throws Excep
         stateUpdater.add(task4);
 
         verifyRestoredActiveTasks(task2, task1);
+        verifyCommitTasks(true, task2, task1);
```

Comment:
What if you move the this verification after `verifyUpdatingStandbyTasks()`? Are standby tasks still not checkpointed because we do not enforce the checkpoint?

Reply:
Yes, unless we wait for commit interval.. btw I think since this is already covered in other tests we do not need to wait  (and since it's system time we need to wait conservatively to reduce flakiness) again in this test.